plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/InMemoryGlueCache.java

```diff
@@ -49,15 +49,21 @@
 import java.util.function.Function;
 import java.util.function.Supplier;
 
+import static io.trino.cache.CacheUtils.invalidateAllIf;
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
 
 class InMemoryGlueCache
         implements GlueCache
 {
-    private enum Global { GLOBAL }
+    private enum Global
+    {
+        GLOBAL
+    }
```

Comment:
I thought we allowed single line enums when there is one value.  Is that not allowed?

Reply:
i can drop this change, if you want. but i want intellij formatter to not change the code, so if it does, we either reformat the code or update the airlift codestyle